use_gnu is set with gnu source and for other things,
so force the xsi fallback
--
diff --git a/src/internal.c b/src/internal.c
index ef83b71..5fae2e4 100644
--- a/src/internal.c
+++ b/src/internal.c
@@ -431,7 +431,7 @@ char *cleanpath(char *path) {
 
 const char *xstrerror(int errnum, char *buf, size_t len) {
 #ifdef HAVE_STRERROR_R
-# ifdef __USE_GNU
+# if 0
     /* Annoying linux specific API contract */
     return strerror_r(errnum, buf, len);
 # else
